code/modules/research/designs/bot_upgrades.dm

```diff
@@ -0,0 +1,43 @@
+///////////////////
+// RESEARCH ITEM //
+///////////////////
+
+/datum/design/bot_upgrade
+	// Should not be acquirable
+
+/datum/design/bot_upgrade/boost
+	name = "Bot Upgrade: Boost"
+	desc = "The circuit board for giving a speed boost to bots"
+	id = "bot_upgrade_boost"
+	req_tech = list("programming" = 2, "engineering" = 2)
+	build_type = MECHFAB
+	materials = list(MAT_GLASS = 500)
+	build_path = /obj/item/weapon/bot_upgrade/boost
+	category = list("Misc")
+	construction_time = 100
+
+////////////////////////
+// CIRCUIT BOARD ITEM //
+////////////////////////
+
+/obj/item/weapon/bot_upgrade/
+	density = 0
```

Comment:
A lot of this file was copy and pasted, and I'm not sure what exactly some of these properties do